dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.options/server.xml

```diff
@@ -5,5 +5,5 @@
     </featureManager>
 
   	<include location="../fatTestPorts.xml"/>
-
+  	<javaPermission className="java.net.SocketPermission" name="*" actions="resolve"/>
```

Comment:
Same as above - how is the test failing with Java 2 security enabled?

Reply:
Same response as above.